lib/ansible/plugins/action/reboot.py

```diff
@@ -218,6 +218,9 @@ def get_system_boot_time(self, distribution):
             except TypeError as e:
                 raise AnsibleError("Invalid value given for 'boot_time_command': %s." % to_native(e))
 
+        key = 'boot_time_command='
+        boot_time_command = f'echo {key}$({boot_time_command})'
```

Comment:
This should use `_SHELL_SUB_LEFT`/`_SHELL_SUB_RIGHT`/`ECHO` of `self._connection._shell`.

Reply:
Got it, thanks! Fixed with commit c5de620.